doc/lib/mqtt.txt

```diff
@@ -52,7 +53,7 @@ The initial wait time before the first reconnect attempt.
 - `AWS_IOT_MQTT_MAX_RECONNECT_WAIT_INTERVAL` <br>
 The maximum wait time between reconnect attempts.
 
-If a reconnect attempt after `AWS_IOT_MQTT_MAX_RECONNECT_WAIT_INTERVAL` fails, @ref mqtt_function_yield will return `NETWORK_RECONNECT_TIMED_OUT`. There will be no more reconnect attempts. If a reconnect is needed after this based on some external conditions, then the application should call @ref mqtt_function_attempt_reconnect.
+If a reconnect attempt after `AWS_IOT_MQTT_MAX_RECONNECT_WAIT_INTERVAL` fails, @ref mqtt_function_yield will return `NETWORK_RECONNECT_TIMED_OUT`. There will be no more reconnect attempts. If an application still wants to re-connect it should call @ref mqtt_function_attempt_reconnect.
```

Comment:
(Nitpick of missing command): If the application still wants to re-connect \*,\*, it should call

Reply:
Fixed